mscore/inspector/inspectorBase.cpp

```diff
@@ -377,23 +384,7 @@ void InspectorBase::valueChanged(int idx, bool reset)
 
 void InspectorBase::resetClicked(int i)
       {
-      Element* e   = inspector->element();
```

Comment:
@wschweer That's a lot of code that you wrote removed here. Could you review this change?

Reply:
@wschweer reviewed "looks ok but i am just wondering if there is a better method to handle multiple selections..."
I'll merge.